modules/installation-ibm-z-troubleshooting-and-debugging.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-ibm-z-troubleshooting-and-debugging_{context}"]
+= Completing installation on user-provisioned infrastructure
+
+You can gather the debugging information that might help you to troubleshoot and
+debug certain issues with {product-title} installation on IBM Z.
+
+.Prerequisites
+
+* The `oc` CLI tool installed.
+
+.Procedure
+
+. Log into the cluster:
++
+----
+$ oc login
+----
+
+. On the node you want to gather hardware information about, start a debugging
+container:
++
+----
+$ oc debug node/<nodename>
+----
+
+. Change to the */host* file system and start `toolbox`:
++
+----
+$ chroot /host
+$ toolbox
+----
+
+. Collect the `dbginfo` data:
++
+----
+$ dbginfo.sh
+----
+
+. You get then retrieve the data, for example, using `scp`.
```

Comment:
This sentence is confusing. Should it be "You can then retrieve..."?

Reply:
Yep, fixed.